models.py

```diff
@@ -2,27 +2,32 @@
 from statsmodels.tsa.arima_model import ARIMA
 import warnings
 import numpy
-def status_quo_model(X):
+
+def status_quo_model(X,data):
     """Predict the most recent value of each series.
 
-    If no most recent value exists, predict 0."""
+    If no most recent value exists, predict mean of all most recent values for the given indicator."""
 
     def mostrecent(x):
         x = x[x.notnull()]
         if len(x) > 0:
             return x.iloc[-1]
         else:
-            return 0
+            df = data._train.copy()
```

Comment:
`data._train` is mean to be a private variable so we shouldn't access it directly. The current preprocessing functions provided by `dataset` don't really give you another option, but I think the better choice here would be to add a new preprocessing function to `dataset` that does what you want.